pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -511,7 +511,7 @@ type Cloud struct {
 	// attached, to avoid a race condition where we assign a device mapping
 	// and then get a second request before we attach the volume
 	attachingMutex sync.Mutex
-	attaching      map[types.NodeName]map[mountDevice]awsVolumeID
+	attaching      map[types.NodeName]map[mountDevice]VolumeID
```

Comment:
The challenge here is that (1) we want to be very clear that it is an AWS volume, not a k8s volume, and (2) that we don't want to export it.

How about `ebsVolumeID`?

Reply:
The lint error this is fixing is that returning an un-exported type (from `MapToAWSVolumeID`) can be annoying to deal with, so it's necessary to export the type to resolve the error. I've considered just not exporting `MapToAWSVolumeID` but it's used in [pkg/volume/awsebs/aws_util.go](kubernetes/kubernetes/blob/master/pkg/volume/awsebs/aws_util.go#L230) and I don't see a great way to break that dependency so decided to export `VolumeID` instead.

How about `EBSVolumeID`?

Reply:
Ah I see.  Yes, EBSVolumeID or AWSVolumeID would both sound good to me, but I presume AWSVolumeID would fall-foul of the "repeating the package name" rule.